open-metadata-implementation/access-services/data-engine/data-engine-client/src/main/java/org/odpi/openmetadata/accessservices/dataengine/client/DataEngineImpl.java

```diff
@@ -95,31 +97,32 @@ public DataEngineImpl(String serverName, String serverPlatformRootURL, String us
     }
 
     @Override
-    public String createProcess(String userId, String qualifiedName, String processName, String description,
-                                String latestChange, List<String> zoneMembership, String displayName, String formula,
-                                String owner, OwnerType ownerType, List<PortImplementation> portImplementations,
-                                List<PortAlias> portAliases, List<LineageMapping> lineageMappings) throws
-                                                                                                   PropertyServerException,
-                                                                                                   InvalidParameterException,
-                                                                                                   UserNotAuthorizedException {
-        final String methodName = "createProcess";
+    public String createOrUpdateProcess(String userId, String qualifiedName, String processName, String description,
```

Comment:
As commented on DataEngineClient, maybe worth update semantics on all of these methods (?)